go.mod: github.com/containerd/typeurl v1.0.2

thaJeztah opened the pull request:
full diff: https://github.com/containerd/typeurl/compare/v1.0.1...v1.0.2

- Fix lock for getTypeByUrl

no _significant_ other changes, other than getting rid of github.com/gogo/protobuf v1.3.1 in the go mod tree

dims commented:
@thaJeztah yea, let's make sure the containerd 1.5.0 RC has tagged versions, if we really don't need to ... then we can wait to get this in

thaJeztah commented:
> @thaJeztah yea, let's make sure the containerd 1.5.0 RC has tagged versions, if we really don't need to ... then we can wait to get this in

Yes, I had a draft 'note' / working on a list of dependencies that have tagged release, but for which we're currently using versions from "tip" that we should tag (or request to be tagged) before GA

thaJeztah commented:
rebased to trigger CI (which should now be fixed)

thaJeztah commented:
Interesting; new failure in script validation?

```
==> default: Checking if box 'fedora/33-cloud-base' version '33.20201019.0' is up to date...
==> default: Running provisioner: install-runc (shell)...
    default: Running: inline script
    default: /tmp/vagrant-install-runc: line 5: GOPATH: unbound variable
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

dmcgowan commented:
v1.0.2 is tagged, update

thaJeztah commented:
Updated; I also removed the second commit that tried to fix the runc installation - as I'm wondering if it was just a flaky (if it's failing again, I'll add it back)

thaJeztah commented:
```
Network Project: cri-c8d-pr-node-e2e
Zone: us-central1-f
Dumping logs from master locally to '/logs/artifacts'
Trying to find master named 'bootstrap-e2e-master'
Looking for address 'bootstrap-e2e-master-ip'
ERROR: (gcloud.compute.addresses.describe) Could not fetch resource:
 - The resource 'projects/cri-c8d-pr-node-e2e/regions/us-central1/addresses/bootstrap-e2e-master-ip' was not found
```

thaJeztah commented:
/test pull-containerd-node-e2e
